database/things/php.pldb

```diff
@@ -100,6 +100,7 @@ features
   /* A comment
   */
  hasSemanticIndentation false
+ hasCaseInsensitiveIdentifiers true
```

Comment:
https://en.wikipedia.org/wiki/Case_sensitivity#In_programming_languages says: 
> in [PHP](https://en.wikipedia.org/wiki/PHP), for example, variable names are case-sensitive but function names are not case-sensitive

Reply:
So @celtic-coder , do we mark it false in this case?

Reply:
Yes, "false" for PHP.